boxsdk/client/client.py

```diff
@@ -3,24 +3,34 @@
 from __future__ import unicode_literals
 import json
 
-from .auth import DeveloperTokenAuth
-from .config import API
-from .session.box_session import BoxSession
-from .network.default_network import DefaultNetwork
-from .object.user import User
-from .object.folder import Folder
-from .object.search import Search
-from .object.events import Events
-from .object.file import File
-from .object.group import Group
-from .object.group_membership import GroupMembership
-from .util.shared_link import get_shared_link_header
-from .util.translator import Translator
+from ..auth import OAuth2
+from ..config import API
+from ..session.box_session import BoxSession
+from ..network.default_network import DefaultNetwork
+from ..object.user import User
+from ..object.folder import Folder
+from ..object.search import Search
+from ..object.events import Events
+from ..object.file import File
+from ..object.group import Group
+from ..object.group_membership import GroupMembership
+from ..util.shared_link import get_shared_link_header
+from ..util.translator import Translator
+from ..util.url import get_url
 
 
 class Client(object):
 
-    def __init__(self, oauth, network_layer=None, session=None):
```

Comment:
I guess it's just a preference. One of the goals for the SDK is making it simple to get started. I've always disliked that you need to import OAuth2 and Client.

But I'm not married to this change. I can remove it for now and we can revisit.